R/bundle.R

```diff
@@ -139,48 +104,63 @@ inferAppMode <- function(appDir, files) {
     return("rmd-static")
   }
 
-  # no Shiny .R files and no R Markdown docs--app mode is unknown
+  # if there are one or more HTML documents, use the static mode
+  htmlFiles <- grep("^[^/\\\\]+\\.html$", files, ignore.case = TRUE,
```

Comment:
We certainly could (I can't vouch that the whole toolchain never produces
.htm and it seems likely that it does in at least one place). I'd
definitely support making the "l" optional here.

On Wed, Apr 8, 2015 at 11:11 AM, Aron Atkins <notifications@github.com>
wrote:

> In R/bundle.R
> <https://github.com/rstudio/rsconnect/pull/27#discussion_r27978925>:
>
> > @@ -139,48 +104,63 @@ inferAppMode <- function(appDir, files) {
> >      return("rmd-static")
> >    }
> >
> > -  # no Shiny .R files and no R Markdown docs--app mode is unknown
> > +  # if there are one or more HTML documents, use the static mode
> > +  htmlFiles <- grep("^[^/\\\\]+\\.html$", files, ignore.case = TRUE,
>
> Do we ever see the .htm extension?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rstudio/rsconnect/pull/27/files#r27978925>.
>